toolchain/check/import_cpp.cpp

```diff
@@ -1950,6 +1890,85 @@ static auto ImportNameDeclIntoScope(Context& context, SemIR::LocId loc_id,
                                                            access_kind);
 }
 
+// Imports an overloaded function set from Clang to Carbon.
+static auto ImportCppOverloadSet(Context& context, SemIR::NameScopeId scope_id,
+                                 SemIR::NameId name_id,
+                                 const clang::UnresolvedSet<4>& overload_set)
+    -> SemIR::InstId {
+  SemIR::CppOverloadSetValue overload_set_inst = {
+      .type_id = SemIR::TypeId::None,
+      .overload_set_id = SemIR::CppOverloadSetId::None};
+
+  overload_set_inst.overload_set_id = context.cpp_overload_sets().Add(
+      SemIR::CppOverloadSet{.name_id = name_id,
+                            .parent_scope_id = scope_id,
+                            .candidate_functions = overload_set});
+
+  overload_set_inst.type_id = GetCppOverloadSetType(
+      context, overload_set_inst.overload_set_id, SemIR::SpecificId::None);
+
+  auto overload_set_inst_id =
+      // TODO: Add a location.
+      AddInstInNoBlock(context, Parse::NodeId::None, overload_set_inst);
```

Comment:
How about instead of creating an empty instruction and then override it, create it as final. Then we can also combine it to add it immediately and only have the `inst_id` as a variable and not the inst.

```suggestion
  SemIR::CppOverloadSetId overload_set_id = context.cpp_overload_sets().Add(
      SemIR::CppOverloadSet{.name_id = name_id,
                            .parent_scope_id = scope_id,
                            .candidate_functions = overload_set});
                            
  auto overload_set_inst_id =
      // TODO: Add a location.
      AddInstInNoBlock<SemIR::CppOverloadSetValue>(context, Parse::NodeId::None, {.type_id = GetCppOverloadSetType(
      context, overload_set_id, SemIR::SpecificId::None),
      .overload_set_id = overload_set_id});
```

Reply:
Done.